features/disabled_bugsnag.feature

```diff
@@ -2,4 +2,5 @@ Feature: Bugsnag disabled
 
 Scenario: No requests are received when bugsnag is disabled
     When I build "default_app" using the "disabled_bugsnag" bugsnag config
-    Then I should receive 0 requests
+    And I wait for 5 seconds
```

Comment:
There are a couple of places which now have waits - why are they necessary?

Reply:
They're needed to make sure we don't have a "late" payload coming through.  If we don't expect a payload, without a wait the likelihood of false positive passes increases greatly.

Reply:
I think we tend to think about this more when expecting 0 requests, but not 1 or more (where the same issue of receiving too many requests could happen).  I guess we just consider 0 to be a special case more worthy of waiting for?  We do already have a ticket to address tests with no positive outcome, perhaps by capturing logs instead, so we can probably tolerate this for now.  I did think that we could put some functionality into Maze Runner itself to wait in just the 0 request case, but that's probably compounding the issue.

Reply:
fair enough - I'm happy with adding a wait if it's necessary for the test and there are plans to rewrite tests which expect 0 requests anyway